x-pack/plugins/ingest_manager/common/types/epm.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// Follow pattern from https://github.com/elastic/kibana/pull/52447
+// TODO: Update when https://github.com/elastic/kibana/issues/53021 is closed
+import {
+  SavedObject,
+  SavedObjectAttributes,
+  SavedObjectReference,
+} from '../../../../../src/core/public';
+import { AssetType as IngestAssetType } from '../../../../legacy/plugins/ingest/common/types/domain_data';
```

Comment:
can we move `AssetType` definition from the old plugin into this file? I'd love to decouple dependencies from the old plugin ASAP 😊

Reply:
I copied this type over, instead of moved, as other types in domain_data reference it. I thought of moving all the types over but didn't want to break Fleet, so I've left them for now.